examples/net_framework/CSharpExamples/DeepSpeechConsole/Program.cs

```diff
@@ -25,7 +25,7 @@ static void Main(string[] args)
                 stopwatch.Start();
                 try
                 { 
-                    result = sttClient.CreateModelAsync("output_graph.pbmm", N_CEP, N_CONTEXT, "alphabet.txt", BEAM_WIDTH).GetAwaiter().GetResult();
+                    result = sttClient.CreateModel("output_graph.pbmm", N_CEP, N_CONTEXT, "alphabet.txt", BEAM_WIDTH);
```

Comment:
nit: shouldn't we use CLI args somehow ? otherwise people get forced into naming files

Reply:
Ok, I'll add args and keep static names for VS Studio runs